mu-plugins/websub-compat.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Plugin Name: WebSub Backwards Compatibility
+ * Version: 1.0.0
+ * Author: WordPress.com Special Projects
+ * Author URI: https://wpspecialprojects.wordpress.com/
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	return;
+}
+
+/**
+ * Filter `pubsubhubbub_feed_urls` and `pubsubhubbub_comment_feed_urls`.
+ *
+ * Still send pings for the old `blog.opensource.org` Feed-URLs.
+ *
+ * @param array $links Array of Feed URLs.
+ *
+ * @return array $links Array of filtered Feed URLs.
+ */
+function websub_compat( array $links ) {
+	if ( ! $links || ! is_array( $links ) ) {
```

Comment:
Do we still need to check for both of these things given the `array` type declaration in the function?

Reply:
Good point! (I added the type hint after PHPCS throw an error and hadn't looked into the rest of the code 🫣)